<?xml version="1.0" encoding="utf-8" ?>
<launch>
    <arg name="topics" default="['/tf', '/point_cloud2', '/image']"/>
    <node name="seerep_ros_communication_client" pkg="seerep_ros_communication" type="seerep_ros_communication_client" output="screen">
        <rosparam param="topics" subst_value="True">$(arg topics)</rosparam>
        <param name="mapFrame" type="string" value="world"/>
    </node>
</launch>
